tests/consumer/test_oauth2.py

```diff
@@ -90,6 +91,43 @@ def test_login_url():
     assert location.query_dict["state"] == "random-string"
 
 
+@responses.activate
+def test_login_url_with_pkce():
```

Comment:
Thanks for adding these tests! We also need to test the `authorization_url` as well, though. Check the test coverage to be sure you've added tests to cover all the lines you added.

Reply:
Yes, I was aware those tests were missing. I will add them for sure.